eos/b-decays/observables.cc

```diff
@@ -56,6 +57,32 @@ namespace eos
 
         return ObservableGroup(imp);
     }
+
+    ObservableGroup
+    make_b_to_3l_nu_group()
+    {
+        auto imp = new Implementation<ObservableGroup>(
+            R"(Observables in $B^-\to \ell_\prime^+\ell_\prime^-\ell^-\bar\nu$ decays)",
+            R"(The option "lprime" selects the lepton flavour coming out of
+            the virtual photon, "l" selects the charged lepton flavour coming
+            out of the weak current.)",
+            {
+                make_observable("B_u->mumuenu::d2BR/dq2/dk2", R"(d\mathcal{B}(B^- \to \mu^+\mu^-e^-\bar\nu)/dq^2/dk^2)",
+                        Unit::None(),
+                        &BToThreeLeptonsNeutrino::branching_ratio_2diff,
+                        std::make_tuple("q2", "k2"),
+                        Options{ { "lprime", "mu" }, { "l", "e" } }),
+
+                make_observable("B_u->mumuenu::d5BR/dq2/dk2/dOmega", R"(d\mathcal{B}(B^- \to \mu^+\mu^-e^-\bar\nu)/dq^2/dk^2/dOmega)",
+                        Unit::None(),
+                        &BToThreeLeptonsNeutrino::branching_ratio_5diff,
+                        std::make_tuple("q2", "k2", "cos_theta_gamma", "cos_theta_w", "phi"),
+                        Options{ { "lprime", "mu" }, { "l", "e" } }),
+            }
```

Comment:
- please rename to ``B_u->enumumu::d2BR/dq2/dk2`` and adjust the LaTeX representation
- please remove the 5-differential rate as an observable *here*
- please add the 2-differential rate for ``B_u->munuee``